docs/design-proposals/multi-tenancy/multi-tenancy-network.md

```diff
@@ -70,7 +72,7 @@ spec:
 And here is a sample of a VPC network:
 
 ```yaml
-apiVersion: v1
+apiVersion: arktos.futurewei.com/v1
 kind: Network
 metadata:
   name: vpc-1
```

Comment:
The vpc-1 is the network name for arktos network object OR Mizar vpc object?  

Reply:
it is Arktos network object name